bundles/org.openhab.binding.pushover/src/main/java/org/openhab/binding/pushover/internal/handler/PushoverAccountHandler.java

```diff
@@ -94,6 +105,15 @@ public Collection<Class<? extends ThingHandlerService>> getServices() {
         return SUPPORTED_THING_ACTIONS;
     }
 
+    @Override
+    public void dispose() {
+        try {
+            this.httpClient.stop();
```

Comment:
Won't this stop the client for all handler instances, in case more than one is configured?